netstandard/ref/System.Net.Http.cs

```diff
@@ -234,6 +240,32 @@ public sealed partial class ReadOnlyMemoryContent : System.Net.Http.HttpContent
         protected override System.Threading.Tasks.Task SerializeToStreamAsync(System.IO.Stream stream, System.Net.TransportContext context) { throw null; }
         protected internal override bool TryComputeLength(out long length) { length = default(long); throw null; }
     }
+    public sealed partial class SocketsHttpHandler : System.Net.Http.HttpMessageHandler
```

Comment:
@davidsh @karelz should we expose SocketsHttpHandler in the next version of .NET Standard? IIRC it is intended to be a temporary bridge API and as such we might not want to standardize it.

Reply:
We should not add it to .NET Standard.  We expect most devs to use HttpClientHandler which will use SocketsHttpHandler underneath.

Reply:
What's the current status here?

Reply:
Sounds good. I've excluded `SocketsHttpHandler`.